.github/workflows/Pipeline.yml

```diff
@@ -1,274 +1,106 @@
-name: Unit Testing, Coverage Collection, Package, Release, Documentation and Publish
+name: Pipeline
 
-on: [ push ]
+on:
+  push:
+  workflow_dispatch:
 
 defaults:
   run:
     shell: bash
 
 jobs:
-  UnitTesting:
-    name: ${{ matrix.icon }} Unit Tests using Python ${{ matrix.python }}
-    runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - {python: 3.6, icon: 🔴}
-          - {python: 3.7, icon: 🟠}
-          - {python: 3.8, icon: 🟡}
-          - {python: 3.9, icon: 🟢}
+  Params:
+    uses: pyTooling/Actions/.github/workflows/Parameters.yml@r0
+    with:
+      name: pyEDAA.CLITool
+      python_version: '3.9'
+      python_version_list: '3.6 3.7 3.8 3.9'
```

Comment:
Please use default and also enable 3.10.

Reply:
Done.